jigar/hangman/hangman.rb

```diff
@@ -0,0 +1,45 @@
+SECRET_WORD = "aaabbbcccc"
+$word = SECRET_WORD.clone

```

Comment:
avoid using global ,variables

Reply:
whiile passing in methods it creates problem so i have to use the global variables